scripts/create.js

```diff
@@ -0,0 +1,24 @@
+export function count() {
+  const createAddQuestion = document.querySelector(
+    "[data-js=text-addquestion]"
+  );
+  const amountLeftAddQuestion = document.querySelector(
+    "[data-js=create-amount-add-question]"
+  );
+  const createAddAnswer = document.querySelector("[data-js=text-addanswer]");
+  const amountLeftAddAnswer = document.querySelector(
```

Comment:
Looks all fine to me; just thought about creating a utility function for document.querySelector. This could make it even more readable. It is totally understandable the way it is, though.